feeds/combiner.go

```diff
@@ -152,14 +161,17 @@ func (cf *combinerFeed) forEachAlert(alertHandlers []cfHandler) error {
 
 		lowerBound := time.Minute * 10
 		upperBound := int64(0)
+
 		// query all subscriptions and push
 		for _, subscription := range cf.Subscriptions() {
+			logger := log.WithField("bot-id", subscription.Subscriber.BotID)
```

Comment:
`forta-node` prefers `botId` at the moment and if we grep by `botId`, we won't be catching both of the logs. Can we prefer camel case in log field names?

Reply:
fixed with, 8104fd66751c5d5cb6317c1f72b14d179f6e45c1